### PR TITLE
yes: add French translation, add common usage example

### PR DESCRIPTION
**Pull request by Hyftar**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

This PR adds a French translation of the `yes` command and a new usage example to illustrate a common use.

### CODE REVIEW

**Comment by schneiderl** [comment]
Hi,
I believe it would be better to have those changes separated into two different commits and two different PRs. 
The changes have no relation between them: one is changing an English page and the other is adding a French page.

No worries though, the changes are small so I believe we can continue with the review process within this pull request. Just take that in my for future PRs.

Also, thank you for your contribution, it looks great to me (even though I don't speak French). 
Be welcome to keep contributing to tldr, it is much appreciated.

**Review by schneiderl — APPROVED** [review]

**Comment by SethiPandi** [comment]
Just my opinion, I think it's great to have an example that uses `yes` via a pipe (`|`), but is `apt-get` really the best example for this?

`apt-get` already has a built-in `-y` argument which is much safer to use than piping from `yes`.

That makes the example a bit redundant, and very unsafe compared to the built-in `-y` argument.
If the point of this example is to say yes automatically, even to unsafe operations, it might be good to document that it will do so?

```
-y, --yes, --assume-yes
           Automatic yes to prompts; assume "yes" as answer to all prompts and
           run non-interactively. If an undesirable situation, such as
           changing a held package, trying to install a unauthenticated
           package or removing an essential package occurs then apt-get will
           abort. Configuration Item: APT::Get::Assume-Yes.
```

**Comment by Hyftar** [comment]
I agree that `apt-get` might not be the safest choice, it's just the first that came to my mind. If you have another command that we could use that would be safer for the average user I would gladly change it in my examples.

**Comment by SethiPandi** [comment]
I've been giving it some thought, but I can't really think of a good example for this that wouldn't be unsafe, or that there is a built-in argument to handle that case. In cases where there isn't a `-y` or equivalent, it still might not be the best idea to spam `y` without contextualizing the prompts.

I was just _especially_ hesitant on this particular example as I'm wary it might encourage poor practices to users that are new to `apt-get` if they aren't aware of the `-y` argument yet. (Which is also why I thought documented that in the example might be nice.)

If anyone else has a nice example, that'd be great, otherwise I feel just a note on the matter might suffice personally.

---

I actually tested this your example a bit locally though and just want to check, are you sure the command isn't meant to be:

```bash
yes | sudo apt-get install {{program}}
```
> This responds `y` to all interactive prompts. I'm succesfully able to install software this way.

Not:
```bash
sudo apt-get install {{program}} | yes
```
> This just hides the command input and spams `y` until killed. The software does not get installed.

Another test I did was with `npm init`.

```bash
yes "" | npm init
```
> This responds with nothing to all prompts, and generates the default `package.json` file.

Not:
```bash
npm init | yes ""
```
> This just hides the command input and spams `y` until killed. The `package.json` does not get created.

**Comment by Hyftar** [comment]
Oh you are indeed right, I'll fix both points shortly. Thanks you for your attention to details!

**Comment by sbrl** [comment]
Perhaps such an example would be best put on the page for `yes`?

**Review by sbrl — COMMENTED** [review]
Oh oops! I  see what those comments were about now.

I don't see anything wrong with this, since there isn't much else to the `yes` command and this is one of it's intended use-cases. `apt-get` does have the `-y` argument though, so I'm unsure as to whether it's the best example. If there isn't another alternative though this will be fine.

**Review by Deuchnord — CHANGES_REQUESTED** [review]
Nice translation!

**Review by sbrl — APPROVED** [review]

**Comment by sbrl** [comment]
Also, next time, perhaps you could avoid force-pushing please? It makes it harder to tell what's changed when reviewing, and we've got a special button that allows us to squash easily on merge :smiley_cat:

**Comment by Hyftar** [comment]
> next time, perhaps you could avoid force-

I only forced push my last commit because it was missing a file. 🙂

**Comment by sbrl** [comment]
Oh, no worries then :-)

File: pages/common/yes.md [hunk 1]
```diff
@@ -3,14 +3,14 @@
 > Output something repeatedly.
 > This command is commonly used to answer yes to every prompt by install commands (such as apt-get).
 
-- Repeatedly output "message":
+- Repeatedly output "message" :
```
Comment (Deuchnord) [inline pages/common/yes.md:6]: I think you have added a space that is not required here ^^

Comment (navarroaxel) [reply pages/common/yes.md:6]:
```suggestion
- Repeatedly output "message":
```

**Review by owenvoke — APPROVED** [review]
I guess people are more likely to use the `-y` flag for `apt-get`, but makes sense as an example of the usage. 👍🏻 Nice work.

**Review by sbrl — APPROVED** [review]

**Pull request merged**